Composer/packages/client/src/pages/publish/Publish.tsx

```diff
@@ -138,59 +162,60 @@ const Publish: React.FC<RouteComponentProps<{ projectId: string; targetName?: st
       dataTestid: 'publishPage-Toolbar-Pull',
       disabled: !isPullSupported,
     },
-    {
-      type: 'action',
-      text: formatMessage('See Log'),
-      buttonProps: {
-        iconProps: {
-          iconName: 'ClipboardList',
-        },
-        onClick: () => setShowLog(true),
-      },
-      align: 'left',
-      disabled: selectedVersion ? false : true,
-      dataTestid: 'publishPage-Toolbar-Log',
-    },
-    {
-      type: 'action',
-      text: formatMessage('Rollback'),
-      buttonProps: {
-        iconProps: {
-          iconName: 'ClipboardList',
-        },
-        onClick: () => rollbackToVersion(selectedVersion),
-      },
-      align: 'left',
-      disabled: selectedTarget && selectedVersion ? !isRollbackSupported(selectedTarget, selectedVersion) : true,
-      dataTestid: 'publishPage-Toolbar-Log',
-    },
   ];
-
-  const onSelectTarget = useCallback(
-    (targetName) => {
-      const url = `/bot/${projectId}/publish/${targetName}`;
-      navigateTo(url);
-    },
-    [projectId]
-  );
-
-  const getUpdatedStatus = (target) => {
+  const getUpdatedStatus = (target, botProjectId) => {
     if (target) {
       // TODO: this should use a backoff mechanism to not overload the server with requests
       // OR BETTER YET, use a websocket events system to receive updates... (SOON!)
       setTimeout(async () => {
-        getPublishStatus(projectId, target);
+        getPublishStatus(botProjectId, target);
       }, 10000);
```

Comment:
move the 10000 to a constant and appropriate variable name

Reply:
Anything associated with polling i would use useInterval hook in hooks.ts instead of setTimeouts in component. Check for usages of that hook in our codebase.